components/listing/Listing.tsx

```diff
@@ -30,9 +30,27 @@ const Listing: React.FC<ListingProps> = ({ tasks = [] }) => {
               Sortowanie
             </h3>
           </div>
-          <div className="grid grid-cols-1 row-gap-8 sm:grid-cols-2 sm:col-gap-5 sm:row-gap-6 md:grid-cols-3 lg:col-span-3">
+          <div className="grid grid-cols-1 row-gap-8 sm:grid-cols-2 sm:col-gap-5 sm:row-gap-6 md:grid-cols-3 lg:col-span-3 ml-8 mr-8 mb-8">
             {tasks.map((task) => (
-              <div key={task._id}>{task._id}</div>
+              <section className=" bg-gray-100 rounded-md" key={task._id}>
```

Comment:
wrzuć narazie do `components/listing/Card.tsx` zobaczymy na ile będzie to reużywalny komponent później

Reply:
zrobione.